common/src/configuration/bootstrap.rs

```diff
@@ -128,6 +130,7 @@ impl Default for ConfigBootstrap {
             input_file: None,
             command: None,
             clean_orphans_db: false,
+            enable_mining: false,
```

Comment:
This doesn't seem to be hooked up to anything. Does this work?

You can also achieve this by setting the env var `TARI_BASE_NODE__RIDCULLY__ENABLE_MINING`

Reply:
It is hooked up on line 153-157 of main.rs
```
    // enable-mining argument takes precedence over config setting
    if bootstrap.enable_mining {
        node_config.enable_mining = true;
    }
```